src/Umbraco.Core/Constants-PropertyEditors.cs

```diff
@@ -90,12 +90,17 @@ public static class Aliases
                 /// ListView.
                 /// </summary>
                 public const string ListView = "Umbraco.ListView";
-                
+
                 /// <summary>
                 /// Media Picker.
                 /// </summary>
                 public const string MediaPicker = "Umbraco.MediaPicker";
 
+                /// <summary>
+                /// Media Picker v.4.
+                /// </summary>
+                public const string MediaPicker3 = "Umbraco.MediaPicker3";
+
```

Comment:
@nielslyngsoe this is a confusing comment if we someday need to access the `MediaPicker3` constant 🙈
Previous in v7 we had `Umbraco.MediaPicker` and then `Umbraco.MediaPicker` which is v8 was just `Umbraco.MediaPicker` (maybe that was v3? 🤔)

Reply:
Oh, thats a mistake. I originally thought the v8 one was called v.3 but figured out that was v2. so this one must be v.3 :-P